.dockerignore

```diff
@@ -0,0 +1,7 @@
+node_modules
```

Comment:
Ignore files so they wont be used by the docker container, especially the env files since they are automatically loaded by the emulator